distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/CatalogBundle.java

```diff
@@ -176,4 +176,27 @@ public void setupCaching(boolean cachingEnabled) throws IOException {
         Configuration configuration = adminConfig.getConfiguration(RESOURCE_DOWNLOAD_MANAGER_PID, null);
         configuration.update(updatedProperties);
     }
+
+    public void setDownloadRetryDelay(int delay) throws IOException
+    {
```

Comment:
We need to reformat this file with the latest formatter (https://github.com/codice/ddf-support/blob/master/support-checkstyle/src/main/resources/ddf-intellij-code-formatter.jar).